base_upflow/models/res_partner.py

```diff
@@ -80,11 +80,17 @@ def get_upflow_api_post_customers_payload(self):
         )
         return payload
 
+    def _get_firstname(self):
+        firstname = ""
+        if self.name:
+            firstname = " ".join(self.name.split())
+        return firstname
+
     def get_upflow_api_post_contacts_payload(self):
         payload = self.prepare_base_payload()
         payload.update(
             {
-                "firstName": self.name,
+                "firstName": self._get_firstname(),
```

Comment:
I'm not sure to understand why this module take care of this (as far i understand this method is used to remove extra spaces) ? could we use the same pattern as bellow is the goald is to avoid sending `False` value ?

```suggestion
                "firstName": self.name or "",
```

Reply:
goal was to send `""` if the name was only composed by `" "` for instance